### PR TITLE
Build fix: Remove dependency on http://europa.eu/rapid/conf/RSS20.xsd and update WorldBank to API to V2

### PR DESCRIPTION
**Pull request by nhirschey**
These are fixes to get the build working again.

1. The dependency on the http://europa.eu/rapid/conf/RSS20.xsd has been creating build problems for awhile. I simply removed the test dependency.

2. Also, [on June 19, 2020, the World Bank deprecated the "v1" api](https://datahelpdesk.worldbank.org/knowledgebase/articles/889386-developer-information-overview).This broke the WorldBank provider and tests. The change to fix this is simple as far as I can tell (add "v2" to the url), but in the new api the indicator queries were failing because of the 1900:2050 date query parameter. I simply changed the date query to "" to minimize code changes. Now the tests pass, and I did some simple exploratory queries and everything seemed to work fine.

### CODE REVIEW

**Comment by zyzhu** [comment]
@dsyme Bump this. This pull is necessary to interact with World Bank data as the old REST API is already gone.

I was planning to update Deedle's documentations. But the docs uses World Bank data as examples and it failed because of this issue.

**Comment by eugbaranov** [comment]
@dsyme @cartermp would be great if you could merge this!

**Comment by cartermp** [comment]
I don't have the ability to approve or merge things. I think the fix does look good though.

**Comment by eugbaranov** [comment]
@cartermp my mistake, sorry!
Maybe @ovatsus or @baronfel could have a look?

**Comment by baronfel** [comment]
Sorry, I don't have merge rights to this repo either. I do concur that this would fix the issue, though.

**Comment by nhirschey** [comment]
For what it's worth, the small change to the worldbank API seems to do fine on the Fslab tutorial, and at least the first few numbers that I've checked match what is on the worldbank [website](https://data.worldbank.org/indicator/SE.TER.ENRR?locations=CZ). See screenshot below.

The other change (deleting the unavailable europa file) was suggested by Don as a fix on another open pull request (https://github.com/fsharp/FSharp.Data/pull/1316), so I went ahead and did it here too.  If the .xsd test that I deleted is required, then perhaps the test could be amended by somebody else in a separate pull request using the .xsd file I referenced docs/content/library/XmlProvider.fsx? 

image

File: src/WorldBank/WorldBankProvider.fs [hunk 1]
```diff
@@ -24,7 +24,7 @@ type public WorldBankProvider(cfg:TypeProviderConfig) as this =
     let asm = System.Reflection.Assembly.GetExecutingAssembly()
     let ns = "FSharp.Data" 
 
-    let defaultServiceUrl = "http://api.worldbank.org"
+    let defaultServiceUrl = "http://api.worldbank.org/v2"
```
Comment (eugbaranov) [inline src/WorldBank/WorldBankProvider.fs:27]: Maybe change to `https` whilst you're at it?

Comment (dsyme) [reply src/WorldBank/WorldBankProvider.fs:27]: lets do in separate PR

**Comment by dsyme** [comment]
@nhirschey Would you like to join me in having admin rights on the repo?

**Comment by nhirschey** [comment]
@dsyme, I am happy join you with admin rights and help where I can.

**Comment by zyzhu** [comment]
@dsyme @cartermp As this pull has been merged, could you find some time to release a newer version on nuget so that the World Bank API can be used again?

I'm trying to update docs on Deedle but some samples using World Bank data are broken because of the retirement of its v1 API. Thanks.

**Comment by cartermp** [comment]
I will try to prepare a fix. Lots of things to add to the release notes!

**Comment by cartermp** [comment]
@zyzhu I have published a 4.0.0 here: https://www.nuget.org/packages/FSharp.Data/4.0.0

Release: https://github.com/fsprojects/FSharp.Data/releases/tag/4.0.0

Please let me know if this is working okay. If not we'll get it sorted.

**Comment by zyzhu** [comment]
That's so quick! It finally works again now. Thanks!

**Pull request merged**